skimage/measure/_label.py

```diff
@@ -1,6 +1,32 @@
+from scipy import ndimage
 from ._ccomp import label_cython as clabel
 
 
+def _label_bool(image, background=None, return_num=False, connectivity=None):
+    """
+    Faster implementation of clabel for boolean input.
+    """
+    from ..morphology._util import _resolve_neighborhood
+    if background == 1:
+        image = ~image
+
+    if connectivity is None:
+        connectivity = image.ndim
+
+    if not 1 <= connectivity <= image.ndim:
+        raise ValueError(
+            "Connectivity below 1 or above %d is illegal."
+            % image.ndim)
```

Comment:
```suggestion
        raise ValueError(
            f'Connectivity for {image.ndim}D image should be in [1, ..., {image.ndim}]. '
            f'Got {connectivity}.'
        )
```

Reply:
Copied from the cython code, I'll fix it there as well. thanks.